src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx

```diff
@@ -321,66 +371,96 @@ class TraceViewHeader extends React.Component<PropType> {
   render() {
     return (
       <HeaderContainer>
-        <ActualMinimap trace={this.props.trace} />
-        <CursorGuideHandler.Consumer>
-          {({displayCursorGuide, hideCursorGuide, mouseLeft, showCursorGuide}) => (
-            <div
-              ref={this.props.minimapInteractiveRef}
-              style={{
-                width: '100%',
-                height: `${MINIMAP_HEIGHT + TIME_AXIS_HEIGHT}px`,
-                position: 'absolute',
-                left: 0,
-                top: 0,
-              }}
-              onMouseEnter={event => {
-                displayCursorGuide(event.pageX);
-              }}
-              onMouseLeave={() => {
-                hideCursorGuide();
-              }}
-              onMouseMove={event => {
-                displayCursorGuide(event.pageX);
-              }}
-              onMouseDown={event => {
-                const target = event.target;
-
-                if (
-                  target instanceof Element &&
-                  target.getAttribute &&
-                  target.getAttribute('data-ignore')
-                ) {
-                  // ignore this event if we need to
-                  return;
-                }
-
-                this.props.dragProps.onWindowSelectionDragStart(event);
-              }}
-            >
-              <MinimapContainer>
-                {this.renderFog(this.props.dragProps)}
-                {this.renderCursorGuide({
-                  showCursorGuide,
-                  mouseLeft,
-                  cursorGuideHeight: MINIMAP_HEIGHT,
-                })}
-                {this.renderViewHandles(this.props.dragProps)}
-                {this.renderWindowSelection(this.props.dragProps)}
-              </MinimapContainer>
-              {this.renderTimeAxis({
-                showCursorGuide,
-                mouseLeft,
-              })}
-            </div>
-          )}
-        </CursorGuideHandler.Consumer>
-        {this.renderSecondaryHeader()}
+        <DividerHandlerManager.Consumer>
+          {dividerHandlerChildrenProps => {
+            const {dividerPosition} = dividerHandlerChildrenProps;
+            return (
+              <React.Fragment>
+                <OperationsBreakdown
+                  style={{
+                    width: `calc(${toPercent(dividerPosition)} - 0.5px)`,
```

Comment:
Why inline styles over props to emotion?

Reply:
This is a perf optimization. This is to bypass emotion from having it to recompute the styles, and re-inserting the generating css into DOM.